File: test_human.mjs
// test_human.mjs
import { launch } from './js/dist/index.js';

const sleep = ms => new Promise(r => setTimeout(r, ms));

const browser = await launch({
    headless: false,
    humanPreset: 'default',
    proxy: 'http://user296388:8cpmj1@89.106.202.176:4586'
});
const page = await browser.newPage();
page.on('dialog', async d => await d.accept());

await page.goto('https://deviceandbrowserinfo.com/are_you_a_bot_interactions', { waitUntil: 'domcontentloaded' });
await sleep(2000);

await page.type('input[type="email"]', 'hello@example.com');
await page.type('input[type="password"]', 'Secure#Pass123');
await page.click('button.btn.btn-primary[type="submit"]');

await sleep(3000);
const text = await page.evaluate(() => document.body.innerText || '');
console.log('\n=== HUMAN RESULT ===');
for (const line of text.split('\n')) {
    const l = line.toLowerCase();
    if (l.includes('isbot') || l.includes('you are') || l.includes('suspicious') || l.includes('superhuman'))
        console.log('  ' + line.trim());
}
await browser.close();

File: test_human.py
# test_human.py
from cloakbrowser_human import launch
import time

browser = launch(
    headless=False,
    human_preset="default",
    proxy="http://user296388:8cpmj1@89.106.202.176:4586"
)
page = browser.new_page()
page.on("dialog", lambda d: d.accept())

page.goto("https://deviceandbrowserinfo.com/are_you_a_bot_interactions")
time.sleep(2)

t = time.time()
print("email...")
page.type('input[type="email"]', "hello@example.com")
print(f"email: {int((time.time()-t)*1000)}ms")

t = time.time()
print("password...")
page.type('input[type="password"]', "Secure#Pass123")
print(f"password: {int((time.time()-t)*1000)}ms")

t = time.time()
print("submit...")
page.click('button.btn.btn-primary[type="submit"]')
print(f"submit: {int((time.time()-t)*1000)}ms")

time.sleep(3)
text = page.evaluate("() => document.body.innerText || ''")
print("\n=== RESULT ===")
for line in text.split("\n"):
    low = line.lower()
    if any(w in low for w in ["isbot", "you are", "suspicious", "superhuman"]):
        print("  " + line.strip())

browser.close()
